docs/bluetooth_fetching_data.md

```diff
@@ -0,0 +1,396 @@
+---
+title: "Fetching Bluetooth Data"
+---
+
+## Choosing a method to fetch data
+
+If the device's primary method to notify of updates is Bluetooth advertisements and its primary function is a sensor, binary sensor, or firing events:
+
+- If all sensors are updated via Bluetooth advertisements: [`PassiveBluetoothProcessorCoordinator`](#passivebluetoothprocessorcoordinator)
+- If active connection are needed for some sensors: [`ActiveBluetoothProcessorCoordinator`](#activebluetoothprocessorcoordinator)
+
+If the device's primary method to notify of update is Bluetooth advertisements and its primary function is not a sensor, binary sensor, or firing events:
```

Comment:
Can we highlight `not`, as it took me quite a while to differentiate this from line 7.

Also, is is `update`, or `updates` like in line 7?

Reply:
adjusted